R/pkg/R/mllib_clustering.R

```diff
@@ -404,11 +411,14 @@ setMethod("summary", signature(object = "LDAModel"),
             vocabSize <- callJMethod(jobj, "vocabSize")
             topics <- dataFrame(callJMethod(jobj, "topics", maxTermsPerTopic))
             vocabulary <- callJMethod(jobj, "vocabulary")
+            trainingLogLikelihood <- callJMethod(jobj, "trainingLogLikelihood")
+            logPrior <- callJMethod(jobj, "logPrior")
```

Comment:
In MLlib, if the model is ```DistributedLDAModel```, it has variables called ```trainingLogLikelihood``` and ```logPrior```. If the model is ```LocalLDAModel```, there is no above variables exist, we can not tell users whether they are numeric. So I'd prefer ```NULL``` to represent not existing.